osbuild/api.py

```diff
@@ -193,6 +201,18 @@ def _cleanup(self):
             os.close(self._output_pipe)
             self._output_pipe = None
 
+def exception(path="/run/osbuild/api/osbuild"):
+    """Send exception to osbuild"""
+    _, exc_value, _ = sys.exc_info()
+    traceback.print_exc()
```

Comment:
This prints the exception-backtrace to stdout, just like the default behavior of python, right? Why don't we re-raise the exception instead of the `sys.exit(2)`?

Reply:
IIRC my idea here was to have a different exit status so that later in `pipeline.py` we could do different things depending on the exit code, something like calling `BuildResult.from_exception(api)` etcpp.